code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -21,6 +21,12 @@
 		mode = SYRINGE_INJECT
 		update_icon()
 
+/obj/item/reagent_containers/syringe/Destroy()

```

Comment:
Use `handle_atom_del` on the syringe gun instead.

Reply:
@AnturK unrelated but why use this over destroy? 

also get a discord smh

Reply:
This is functionality of the syringe gun and that proc is dedicated specifically to stuff like this, it's more oop that way.

Reply:
Didn't know about that proc, much cleaner. Thanks!